### PR TITLE
Make containerd/soci log level configurable

### LINKED ISSUE #426: [FEATURE] Make integration test log level configurable
**Is your feature request related to a problem? Please describe.**
Integration test output is unnecessarily verbose during typical runs.

**Describe the solution you'd like**
Provide a mechanism for specifying a default log level (error, warning, info, debug) that will filter test output (e.g. omitting `log.G().Debug()`) and also be passed to processes that are launched via tests (e.g. `soci -log-level`, `soci-snapshotter-grpc --log-level`, `containerd --log-level`, containerd config debug.level).

**Describe alternatives you've considered**
Manual filtering of test output is time consuming.

### PR DESCRIPTION
**Pull request by djdongjin**
**Issue #, if available:** part of #426

**Description of changes:**

This PR makes the following changes:

1. containerd (config and `containerd --log-level`) log level configurable and `warn` by default.
2. soci-snapshotter (`soci-snapshotter-grpc --log-level`) log level configurable and `debug` by default (no change).
3. add `-q` to `docker(-compose) build` cmd, reduce build output: build produces a lot of output but I didn't find it useful in almost all cases.
4. rename integraion testutils to `util_test.go` and `util_soci_test.go`.

This PR reduces integration test log from 22186 lines to around 10000 lines. (if change soci loglevel to warn, to 7500 lines)

I didn't make a global log level config because I feel we want it different in different components.

**Testing performed:**

make check && make test && make integration

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by sparr** [comment]
While this is a good start, I don't think this should close #426, that needs a lot more coverage.

**Comment by djdongjin** [comment]
> While this is a good start, I don't think this should close https://github.com/awslabs/soci-snapshotter/issues/426, that needs a lot more coverage.

The new revision made soci log level configurable (still debug by default). (tested changing to `warn` locally, the output line reduces to 7320.)

The next thing is `logrus.SetLevel()` but checking the output, there are only a couple of lines with `"level":xxxx` after changes made to containerd/soci. So I don't feel that helps much reduce integration output.

Most of the existing 7320 lines come from **output** not **log** (i.e. `fmt.Printf`/etc). They could possibly be minimized either by caller (e.g., our inte tests figure out a way of ignoring cmd exection output) or the callee (e.g., use the `-q` flag in nerdctl/docker), but there isn't a unified config/way of reducing that.

There are some duplication tracked in #404. Fixing that can reduce another ~1500 lines. (The rest are output).

Any other coverage you have in mind?

File: integration/main_test.go [hunk 1]
```diff
@@ -42,7 +42,26 @@ import (
...
+// this can be overwritten by setting up env variables specified by
+// `containerdLogLevelEnv`/`sociLogLevelEnv`.
+var (
+	containerdLogLevel = "warn"
+	sociLogLevel       = "debug"
+)
+
+func validateLogLevel(level string) bool {
+	switch level {
+	case "trace", "debug", "info", "warn", "error", "fatal", "panic":
```
Comment (Kern--) [inline integration/main_test.go:60]: nit: What do you think about replacing this with a logrus check? 
```
_, err := logrus.ParseLevel(level);
return err == nil;
```

**Review by Kern-- — APPROVED** [review]

File: integration/main_test.go (hunk 1 above)
Comment (djdongjin) [reply integration/main_test.go:60]: sure, not aware of this :) got the list of value from cli help output 😅

**Review by Kern-- — APPROVED** [review]

**Review by turan18 — APPROVED** [review]

**Pull request merged**